src/common/print.ts

```diff
@@ -1,5 +1,7 @@
+import { getGlobalConfig } from './config';
+
 function printLog (text: string) {
-    console.log(text);
+    getGlobalConfig().logger.info(text);
```

Comment:
I think it would be better to use getGlobalConfig().logger(text); Because you can't predict which function will come.

Reply:
I think this is a great idea! I'll implement it ASAP and update the PR...